pkg/kudoctl/kudoinit/setup/setup.go

```diff
@@ -89,7 +89,7 @@ func (i *Installer) PreUpgradeVerify(client *kube.Client, result *verifier.Resul
 
 	// Step 2 - Verify that any migration is possible
 	migrations := requiredMigrations()
-	clog.Printf("Verify that %d required migrations can be applied", len(migrations))
+	clog.V(1).Printf("Verify that %d required migrations can be applied", len(migrations))
```

Comment:
based on this guidance
```
// guidance for use of V level
//  0-1 normal standard out
//  2-4 as debug-level logs
//  5-6 logical chooses
//  7-8 input/output details
//  9-10 as trace-level (http details)
```

I would raise this to 2. But I am not blocking the PR on that :)

Reply:
I'd categorize this as normal standard out - which it was before, so I'm fine with 1. Maybe at some point we can make a pass and adjust all of the logs :D